### PR TITLE
update for GeoNetwork v3.6.0 / docker-compose file

### PR DESCRIPTION
**Pull request by gannebamm**
added 3.6.0 folder with appropriate Dockerfiles and added a example docker-compose.yml file

### CODE REVIEW

**Comment by doublebyte1** [comment]
@gannebamm : it looks god, but could you separate this in two commits? one for the update to the latest release and another for the docker-compose. It makes more sense to keep these separate in the history. Thank you!

File: docker-compose.yml [hunk 1]
```diff
@@ -0,0 +1,61 @@
+# GeoNetwork
```
Comment (doublebyte1) [inline docker-compose.yml]: @tianon @yosifkit Is there any best practice concerning having docker-compose files in an official docker image repository?

File: 3.6.0/postgres/docker-entrypoint.sh [hunk 2]
```diff
@@ -0,0 +1,51 @@
+#!/bin/bash
```
Comment (doublebyte1) [inline 3.6.0/postgres/docker-entrypoint.sh]: @gannebamm This file needs running permissions (chmod +x ), in order to be executed whithin the container.

File: 3.6.0/postgres/Dockerfile [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+RUN sed -i -e 's#<import resource="../config-db/h2.xml"/>#<!--<import resource="../config-db/h2.xml"/> -->#g' $CATALINA_HOME/webapps/geonetwork/WEB-INF/config-node/srv.xml && \
+sed -i -e 's#<!--<import resource="../config-db/postgres.xml"/>-->#<import resource="../config-db/postgres.xml"/>#g' $CATALINA_HOME/webapps/geonetwork/WEB-INF/config-node/srv.xml
+
+COPY ./jdbc.properties $CATALINA_HOME/webapps/geonetwork/WEB-INF/config-db/jdbc.properties
+
+#Initializing database & connection string for GN
+COPY ./docker-entrypoint.sh /entrypoint.sh
+ENTRYPOINT ["/entrypoint.sh"]
+
+CMD ["catalina.sh", "run"]
```
Comment (doublebyte1) [inline 3.6.0/postgres/Dockerfile]: @gannebamm I get some errors while running this image. Not sure if it the bug is on the support to postgres in GN, itself. @juanluisrp ?

File: 3.6.0/docker-entrypoint.sh [hunk 4]
```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
```
Comment (doublebyte1) [inline 3.6.0/docker-entrypoint.sh]: @gannebamm This file needs running permissions (chmod +x ), in order to be executed whithin the container.

File: docker-compose.yml (hunk 1 above)
Comment (yosifkit) [reply docker-compose.yml]: You are welcome to add any to your repo, but they won't really be tracked by us. We do have support for adding a single example `stack.yml` to the [Docs repo](https://github.com/docker-library/docs/) (and you already have [one](https://github.com/docker-library/docs/blob/e4dc26fea4ff449f53837daed1f348363c072423/geonetwork/stack.yml)) and you are welcome to link to others or embed them in the `content.md`.

Comment (doublebyte1) [reply docker-compose.yml]: @yosifkit Thanks, I have merged this and added a PR in the library [here](https://github.com/docker-library/official-images/pull/5347).

File: 3.6.0/postgres/Dockerfile (hunk 3 above)
Comment (juanluisrp) [reply 3.6.0/postgres/Dockerfile]: I'm checking it today, @doublebyte1.

Comment (doublebyte1) [reply 3.6.0/postgres/Dockerfile]: @juanluisrp  The problem should be solved with [this](https://github.com/geonetwork/docker-geonetwork/commit/27b11755dc0fa7fcf5be261c089c4b359a4be304) commit.

**Pull request merged**